Retarus/tests/test_retarus_events_forwarder.py

```diff
@@ -0,0 +1,68 @@
+import copy
+import os
+import time
+from datetime import datetime, timedelta, timezone
+from threading import Thread
+from unittest import mock
+from unittest.mock import Mock
+
+import orjson
+import pytest
+
+from retarus_modules.connector import RetarusConnector
+from tests.data import ORIGINAL_MESSAGE
+from threading import Thread
+
+
+def test_forward_on_message(connector, queue):
+    expected_message = orjson.loads(ORIGINAL_MESSAGE)
+    message = copy.deepcopy(expected_message)
+
+    queue.put(message, block=False)
+    connector.events_queue = queue
+
+    Thread(target=connector.execute).start()
+    time.sleep(0.5)
+    connector.stop()
+
+    for call in connector.push_events_to_intakes.call_args_list:
+        assert call.kwargs["topic"] == "qux"
```

Comment:
Are you sure of this?

Reply:
Mh the test passes because `connector.push_events_to_intakes.call_args_list == []`... so I guess not

Reply:
Should be good now